src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2174,578 @@ definitions:
         description: path to patch
       value:
         type: string
-        description: value of path
\ No newline at end of file
+        description: value of path
+  
+  subscribe_user_to_alert_body:
+    type: object
+    description: The request payload used to subscribe emailIds to query or schedule IDs for a given alert type.
+    required:
+      - assetId
+      - alertType
+      - subscriptions
+    properties:
+      assetId:
+        type: string
+        description: The query or schedule ID that you want the alert to track.
+        example: 'a679dd0e-bcb2-4e69-a610-22d17ba98cac'
+      alertType:
+        type: string
+        description: The state of query execution that you want to trigger an alert. The alert type is one of three values (`start`, `success`, `failure`).
+        example: 'failure'
+      subscriptions:
+        type: object
+        required:
+          - emailIds
+          - inContextNotifications
+          - emailNotifications
+        description: This object provides the users details that you want to receive the alert and how the users want to receive them. You must include their email ID and preferred notification channels for the alerts.
+        properties:
+          emailIds:
+            type: array
+            description: These are Adobe registered email addresses. The maximum number of `emailID`s that can be registered at once is five.
```

Comment:
"A list of email address identifiers for the users you are subscribing. These must be Adobe-registered email addresses. [...]"